internal/cmd/schema-registry/command_schema_describe.go

```diff
@@ -112,7 +112,7 @@ func describeById(id string, srClient *srsdk.APIClient, ctx context.Context) err
 		return err
 	}
 
-	return printSchema(schemaID, schemaString.Schema, schemaString.SchemaType, schemaString.References)
+	return printSchema(schemaID, schemaString.Schema, schemaString.SchemaType, schemaString.References, schemaString.Metadata, schemaString.RuleSet)
```

Comment:
Can we just pass `schemaString`? (Also, it looks like `schemaString` is a `struct`, not a `string`)

Reply:
Can we nest one struct in the other? This seems very weird to me.

Reply:
yeah unfortunately that's returned by the API 
https://github.com/confluentinc/schema-registry-sdk-go/blob/master/api_default.go#L373
vs 
https://github.com/confluentinc/schema-registry-sdk-go/blob/master/api_default.go#L385

Reply:
Confusing how `SchemaString` is not a string... Ok. We can leave this as-is.